Example/PaymentSheet Example/PaymentSheet Example/EmbeddedPlaygroundViewController.swift

```diff
@@ -51,41 +64,97 @@ class EmbeddedPlaygroundViewController: UIViewController {
+             configuration: configuration)
+             embeddedPaymentElement.view.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(embeddedPaymentElement.view)
         self.view.addSubview(checkoutButton)
 
         NSLayoutConstraint.activate([
-            paymentMethodsView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            paymentMethodsView.centerYAnchor.constraint(equalTo: view.centerYAnchor),
-            paymentMethodsView.widthAnchor.constraint(equalTo: view.widthAnchor),
+            embeddedPaymentElement.view.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            embeddedPaymentElement.view.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+            embeddedPaymentElement.view.widthAnchor.constraint(equalTo: view.widthAnchor),
             checkoutButton.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 0.9),
             checkoutButton.heightAnchor.constraint(equalToConstant: 50),
             checkoutButton.centerXAnchor.constraint(equalTo: view.centerXAnchor),
             checkoutButton.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -20),
         ])
     }
+
+    private func setupLoadingIndicator() {
+        view.addSubview(loadingIndicator)
+
+        NSLayoutConstraint.activate([
+            loadingIndicator.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            loadingIndicator.centerYAnchor.constraint(equalTo: view.centerYAnchor),
+        ])
+    }
+
+    private func presentError(_ error: Error) {
+        DispatchQueue.main.async {
+            let alert = UIAlertController(title: "Error",
+                                          message: error.localizedDescription,
+                                          preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "OK", style: .default))
+            self.present(alert, animated: true)
+        }
+    }
 }
 
-extension STPPaymentMethod {
-    static var mockPaymentMethod: STPPaymentMethod? {
-        let amex =
-            [
-                "card": [
-                    "id": "preloaded_amex",
-                    "exp_month": "10",
-                    "exp_year": "2020",
-                    "last4": "0005",
-                    "brand": "amex",
-                ],
-                "type": "card",
-                "id": "preloaded_amex",
-            ] as [String: Any]
-        return STPPaymentMethod.decodedObject(fromAPIResponse: amex)
+extension EmbeddedPaymentElement.Configuration {
+
+    /// Initializes an EmbeddedPaymentElement.Configuration from a given PaymentSheet.Configuration.
+    ///
+    /// - Parameters:
+    ///   - paymentSheetConfig: The PaymentSheet.Configuration instance to convert from.
+    ///   - formSheetAction: The FormSheetAction specific to EmbeddedPaymentElement.Configuration.
+    ///   - hidesMandateText: Determines whether to hide mandate text. Defaults to `false`.
+    public init(
```

Comment:
How will we add Embedded-specific stuff like `hidesMandateText` or `formSheetAction` if we go from playground settings -> PaymentSheet.Configuration -> EmbeddedConfiguration?

Maybe we should make the "playground settings -> EmbeddedConfiguration" logic happen in the PlaygroundController or something?

Reply:
Yeah I think creating the EmbeddedConfiguration in the PlaygroundController sounds good, I was thinking we could just inject those two properties into this view controller but that doesn't scale well if we add more. Will update this in this PR.